### PR TITLE
Bug 1325838 - Stop using Python asserts for conditions that should be checked in production

### PR DESCRIPTION
**Pull request by richardasaurus**
Asserts do nothing in python optimise mode so removed them.

https://bugzilla.mozilla.org/show_bug.cgi?id=1325838

### CODE REVIEW

File: treeherder/embed/views.py [hunk 1]
```diff
@@ -11,13 +11,14 @@ class PushStatusView(TemplateView):
 
     template_name = "embed/push_status.html"
 
-    def get_context_data(self, **kwargs):
-        assert "repository" in kwargs and "revision" in kwargs
-        (repository_name, revision) = (kwargs['repository'], kwargs['revision'])
-        context = super(PushStatusView, self).get_context_data(**kwargs)
+    def get_context_data(self, repository, revision):
```
Comment (richardasaurus) [inline treeherder/embed/views.py]: no test needed for this as already covered by django and would raise `NoReverseMatch` https://github.com/mozilla/treeherder/blob/master/treeherder/embed/urls.py#L10

**Comment by richardasaurus** [comment]
@edmorley

**Comment by edmorley** [comment]
Hi Richard - thank you for the PR. Did you mean to close it?

**Comment by richardasaurus** [comment]
yes

**Pull request closed without merging**